platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -262,73 +284,104 @@ public void initInputDevices() {
 		for (int deviceId : deviceIds) {
 			InputDevice device = inputManager.getInputDevice(deviceId);
 			if (DEBUG) {
-				Log.v("GodotView", String.format("init() deviceId:%d, Name:%s\n", deviceId, device.getName()));
+				Log.v("GodotInputHandler", String.format("init() deviceId:%d, Name:%s\n", deviceId, device.getName()));
```

Comment:
Shouldn't this use the `tag` class field instead of `GodotInputHandler`?

Reply:
yes